File: ProblemSet2/ps2.java
import java.util.Scanner;

public class ps2 {

    public static void main(String[] args) {
        int p, aw = 0, dw = 0;
        Scanner in = new Scanner(System.in);
        p = in.nextInt();
        String w = in.next();
        char ws[] = new char[p];
        for (int i = 0; i < p; i++) {
            ws[i] = w.charAt(i);
        }

        for (int i = 0; i < p; i++) {
            if (ws[i] == 'A') aw++;
            else if (ws[i] == 'D') dw++;
        }
        if (aw > dw) System.out.print("Anton");
        else if (aw < dw) System.out.print("Danik");
        else System.out.print("Friendship");
    }
}

File: ProblemSet2/ps2.cpp
#include <iostream>

using namespace std;

int main(){
    int p, aw=0,dw=0;
    cin >> p;
    string w;
    cin>>w;
    for (char ply:w) {
        if(ply=='A')aw++;
        else if (ply=='D')dw++;
    }

    if(aw > dw) cout << "Anton"<<endl;
    else if (aw <dw) cout << "Danik"<<endl;
    else cout << "Friendship"<<endl; // lul
    return 0;

}
